tests/support/foo-resolve/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+	"version": "1.6.2",
```

Comment:
meh, don't think we need this field at all.  I've built my test cases via copy & paste.